lib/utils/check-version.js

```diff
@@ -0,0 +1,41 @@
+const request = require("request-promise");
+const Logger = require.main.require("lib/logger/logger.js");
+const Errors = require.main.require("lib/response/error.js");
+
+function CheckVersion(pkg) {
```

Comment:
>shouldn't that be using snake_case? :)
We actually don't have any strict function naming convention, so I chose that one.

>If I understand correctly, the pkg argument should be the json-parsed contents of package.json?

Yes. Please check the tests, it's far easier to write tests for it.